assets/css/style.css

```diff
@@ -0,0 +1,1077 @@
+/* Font */
+
+@import url("https://fonts.googleapis.com/css2?family=Bai+Jamjuree:wght@400;500;600;700&display=swap");
+
+/* || RESET */
+*,
+*::after,
+::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+input,
+button,
+textarea {
+  font: inherit;
+}
+
+/* || VARIABLES */
+:root {
+  /* ||FONTS */
+  --FF: "Lato", sans-serif;
+  --FF-HEADINGS: "Lato", sans-serif;
+  --FS: clamp(1rem, 2.2vh, 1.5rem);
+
+  /* || COLORS */
+  --BGCOLOR: orange;
+  --BGCOLOR-FADE: rgb(252, 220, 160);
+  --BGIMAGE: linear-gradient(to bottom, var(--BGCOLOR), var(--BGCOLOR-FADE));
+  --BODY-BGCOLOR: #fff;
+  --FONT-COLOR: #000;
+  --BORDER-COLOR: #333;
+  --HIGHLIGHT-COLOR: rgb(51, 178, 51);
+  --LINK-COLOR: #000;
+  --LINK-HOVER: violet;
+  --LINK-ACTIVE: orange;
+  --HEADER-BGCOLOR: #000;
+  --HEADER-COLOR: #fff;
+  --NAV-BGCOLOR: #fff;
+  --HERO-BGCOLOR: rgba(51, 178, 51, 0.75);
+  --HERO-COLOR: #fff;
+  --C-BRAND: #f83691;
+  --C-BRAND-RGB: 248, 54, 145;
+  --C-DARK: #303050;
+  --C-BODY: #636383;
+  --C-LIGHT: #f6f6f6;
```

Comment:
- [ ] Following the project requirements, the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.
- Basic color swatch: #272A31, #EC5242, #D3D3D3
Kindly work on the changes.